dist/dattobd.spec

```
@@ -547,13 +547,11 @@ rm -rf %{buildroot}
 %config %{_sysconfdir}/sysconfig/modules/dattobd.modules
 %endif
 %dir %{_kmod_src_root}
-%{_kmod_src_root}/Makefile
+%{_kmod_src_root}/*.[c,h]
```

This will not work on debbuild, I think?

Just split this into two lines:

```suggestion
%{_kmod_src_root}/*.c
%{_kmod_src_root}/*.h
```